SpatialGDK/Source/Private/Interop/Connection/SpatialWorkerConnection.cpp

```diff
@@ -40,7 +40,7 @@ void USpatialWorkerConnection::ConnectToReceptionist(bool bConnectAsClient)
 {
 	if (ReceptionistConfig.WorkerType.IsEmpty())
 	{
-		ReceptionistConfig.WorkerType = bConnectAsClient ? TEXT("UnrealClient") : TEXT("UnrealWorker");
+		ReceptionistConfig.WorkerType = bConnectAsClient ? SpatialConstants::ClientWorkerType : SpatialConstants::ServerWorkerType;
 	}
 
 	if (ReceptionistConfig.WorkerId.IsEmpty())
```

Comment:
We need to update the ticket at line 51 below. https://improbableio.atlassian.net/browse/UNR-579